.github/workflows/base.yml

```diff
@@ -0,0 +1,78 @@
+name: Build docker images
+
+on:
+  workflow_dispatch:
+    inputs:
+      SUPPORTED_PLATFORMS:
+        default: 'linux/amd64'
+        type: choice
+        options:
+        - 'linux/amd64'
+        - 'linux/amd64,linux/arm64'
```

Comment:
Using `linux/amd64,linux/arm64'` will currently work only on `base` image, and will not succeed for `common`. This is ok from my perspective as I will not attempt to run incompatible combinations. Can/should we enforce this some other way?

Reply:
I believe it is OK to fail if incorrect combo is used. When image will be ready also `arm` version build will succeed.